packages/eas-cli/src/metadata/utils/array.ts

```diff
@@ -0,0 +1,4 @@
+export function unique<T = any>(items: T[]): T[] {
```

Comment:
we already have uniqBy https://github.com/expo/eas-cli/blob/main/packages/eas-cli/src/utils/expodash/uniqBy.ts

Reply:
`unique` and `uniqBy` seems to be different functions, we don't need the `By` functionality here. We just want whatever is unique, I think we mostly use this to find locales from the config. This `uniqBy` seems to also require a `getKey` method, which would be `v => v` in our case, plus it seems to be slower as it would iterate everything (not sure if that would be noticeable)

Happy to just add a `unique` method to expodash though?

Reply:
I would just use uniqBy, but I'm fine with just moving it to expodash(and renaming to be consistent `uniq`)

Reply:
Done in PR https://github.com/expo/eas-cli/pull/1147